src/components/NavBar/NavBar.js

```diff
@@ -21,12 +22,20 @@ const NavBar = () => {
                         alt="acm logo"
                         onClick={() => navigate("/")} />
                 </div>
-                <div className="element-container">
-                    <DropDownButton
-                        text="Resources"
-                        isExpanded={isMenuExpanded}
-                        onClick={() => setMenuExpanded(prev => !prev)}
-                    />
+                <div className="nav-btn-group">
```

Comment:
I think we can get rid of the `nav-btn-group` div and instead place both the `<DropDownButton/>`  and the `LinkButton/>` into a single `element-container` div. Unless there was a reason for doing it this way that I'm not seeing?